### PR TITLE
W 9611964 to tomcat 8.5.73 with mongo url support

### PR DESCRIPTION
**Pull request by rliEG**
Do we need to upgrade the version number for this change? 
1.6.5-evg5

### CODE REVIEW

File: src/main/java/com/dawsonsystems/session/MongoManager.java [hunk 1]
```diff
@@ -593,9 +485,9 @@ private void initDbConnection(String path) throws LifecycleException {
       }
       db.setWriteConcern(WriteConcern.ACKNOWLEDGED);
       getCollection().createIndex(new BasicDBObject("lastmodified", 1));
-      log.info("Connected to Mongo " + host + "/" + database + " for session storage, slaveOk=" + slaveOk + ", " + (getMaxInactiveInterval() * 1000) + " session live time");
+      log.info("Connected to Mongo " + host + "/" + database + " for session storage, slaveOk=" + slaveOk + ", " + (getSessionMaxAliveTime() * 1000) + " session live time");
     } catch (RuntimeException | IOException e) {
-      e.printStackTrace();
+      log.severe(String.valueOf(e.getStackTrace()));
```
Comment (rliEG) [inline src/main/java/com/dawsonsystems/session/MongoManager.java:490]: Changed per @ispringer 's comments on original PR https://github.com/evergage/Mongo-Tomcat-Sessions/pull/7. 

but is it possible the exception was thrown due to writing logs? In that case, the error could get lost. BTW, the e.printStackTrace() code was from the original code we forked from.

**Comment by rliEG** [comment]
Targeting 1.6.5.x  which was the default branch and where the original branch for upgrading to 8.5.x was cut.

File: src/main/java/com/dawsonsystems/session/MongoManager.java (hunk 1 above)
Comment (ispringer) [reply src/main/java/com/dawsonsystems/session/MongoManager.java:490]: Use https://docs.oracle.com/javase/8/docs/api/java/util/logging/Logger.html#log-java.util.logging.Level-java.lang.String-java.lang.Throwable- instead. Let the logger handle stringifying the exception.

File: src/main/java/com/dawsonsystems/session/MongoManager.java [hunk 2]
```diff
@@ -593,9 +485,9 @@ private void initDbConnection(String path) throws LifecycleException {
       }
       db.setWriteConcern(WriteConcern.ACKNOWLEDGED);
       getCollection().createIndex(new BasicDBObject("lastmodified", 1));
-      log.info("Connected to Mongo " + host + "/" + database + " for session storage, slaveOk=" + slaveOk + ", " + (getMaxInactiveInterval() * 1000) + " session live time");
+      log.info("Connected to Mongo " + host + "/" + database + " for session storage, slaveOk=" + slaveOk + ", " + (getSessionMaxAliveTime() * 1000) + " session live time");
```
Comment (ispringer) [inline src/main/java/com/dawsonsystems/session/MongoManager.java:488]: Include the units of the duration in the log message

**Review by ispringer — CHANGES_REQUESTED** [review]

**Comment by ispringer** [comment]
> Do we need to upgrade the version number for this change? 1.6.5-evg5

It looks like `1.6.5-evg4` has been deployed to prod7 (HIPAA), so, yes, we should increment the version.

**Comment by rliEG** [comment]
The pom has eng5 already before this PR, but if it's not released yet, yep, we can continue use it.  reverted back the severe level logging.

**Review by ispringer — APPROVED** [review]

**Comment by astral303** [comment]
> Do we need to upgrade the version number for this change? 1.6.5-evg5

Yes, this branch must update the version to `1.6.5-evg5`.

**Review by astral303 — CHANGES_REQUESTED** [review]
Should update version to 2.0.0-evg1. And we should make a new branch, 2.0.0.x and switch the default branch to that eventually. Also the hypergage `.strata.yml` file will need to be updated to reflect this change.

**Review by sshresthaEG — APPROVED** [review]
Tested locally with Tomcat 8.5.73

**Pull request merged**